bpython/test/test_import_not_cyclical.py

```diff
@@ -0,0 +1,108 @@
+from bpython.test import unittest
+from bpython.importcompletion import find_modules
+import os, sys, tempfile
+
+
+class TestAvoidSymbolicLinks(unittest.TestCase):
+    def setUp(self):
+        with tempfile.TemporaryDirectory() as import_test_folder:
+            try:
+                os.chdir(import_test_folder)
+            except:
+                print("Can't change the Current Working Directory")
+                import_test_folder.cleanup()
+                sys.exit()
```

Comment:
I don't think that is useful. This should report as test failure instead, so I'd suggest to not catch any of these exceptions.